commons-okhttp/src/test/scala/com/avsystem/commons/okhttp/rest/SomeApi.scala

```diff
@@ -0,0 +1,44 @@
+package com.avsystem.commons
+package okhttp.rest
+
+import com.avsystem.commons.rest.{DefaultRestApiCompanion, GET, Header, POST, Path, Query}
+
+trait SomeApi {
```

Comment:
I'd like to have one REST test inferface for all sorts for tests. Currently, that would be `RestTestApi`. Can you adjust it instead of creating more traits which test more or less the same stuff?